dir.props

```diff
@@ -267,6 +267,7 @@
     <IntermediateOutputRootPath Condition="'$(IntermediateOutputRootPath)' == ''">$(BaseIntermediateOutputPath)$(OSPlatformConfig)\</IntermediateOutputRootPath>
     <IntermediateOutputPath Condition="'$(IntermediateOutputPath)' == ''">$(IntermediateOutputRootPath)$(MSBuildProjectName)\</IntermediateOutputPath>
 
+    <ForPackagingDir>$(IntermediateOutputRootPath)sign/forPackaging/</ForPackagingDir>
```

Comment:
I didn't see that one when I searched for "forPackaging", the important thing to me was to have a common definition somewhere.  I think that makes sense, I'll move it into root\dir.props